zygrader/config/changelog.txt

```diff
@@ -1,5 +1,11 @@
 # Try to keep lines shorter than 70 characters
 
+5.13.0
+* (Admin) add new option to set recently graded durations from
+  zygrader.
+* Fix crash when canceling the "Run Code" option
+* Fix crash in the grade puller with multiple-matched ids
+
```

Comment:
I think this is from the other current feature being pr'd. Nonetheless, it should be updated appropriately before folding changes in (the version number here and elsewhere, and also the message itself)

Reply:
Actually that's from the last version update. Strange that it would show up here...